Examples/Volume/WebXRVolume/index.md

```diff
@@ -0,0 +1,17 @@
+
+### WebXR Volume Viewer
+
+vtk.js supports volume rendering in virtual and augmented reality environments for WebXR compatible systems.
+
+In this example a .vti volume file found [here](https://data.kitware.com/api/v1/file/59de9dca8d777f31ac641dc2/download) is loaded and rendered. XR rendering can be launched on a WebXR-compatible system by pressing the "Enter VR" or "Enter AR" button. Other .vti files can be specified by providiing the `?fileURL=<link>` URL parameter.
```

Comment:
Stay active and present?  

This example loads and renders a .vti volume file found here.  Press the "Enter VR" or "Enter AR" button to launch rendering with WebXR.  To specify other .vti files, provide the `?fileURL=<link>` URL parameter.

Reply:
👍 Will update